com.unity.shadergraph/Editor/Data/Nodes/Input/Basic/ColorNode.cs

```diff
@@ -108,16 +137,29 @@ public override string GetVariableNameForSlot(int slotId)
 
         public override void CollectPreviewMaterialProperties(List<PreviewProperty> properties)
         {
+            UnityEngine.Color propColor = color.color;
+            //why doesn't this seem to matter which colorspace the project is in?
+            if (color.mode == ColorMode.HDR)
+            {
+                switch(version)
+                {
+                    case 0:
+                        break;
+                    case 1:
+                        propColor = propColor.gamma;
+                        break;
+                }
+            }
             properties.Add(new PreviewProperty(PropertyType.Color)
```

Comment:
`


            UnityEngine.Color propColor = color.color;
            if (color.mode == ColorMode.Default)
            {
                if (PlayerSettings.colorSpace == ColorSpace.Linear)
                    propColor = propColor.linear;
            }
            if (color.mode == ColorMode.HDR)
            {
                switch(version)
                {
                    case 0:
                        break;
                    case 1:
                        if (PlayerSettings.colorSpace == ColorSpace.Gamma)
                            propColor = propColor.gamma;
                        break;
                }
            }

            // we use Vector4 type to avoid all of the automatic color conversions of PropertyType.Color
            properties.Add(new PreviewProperty(PropertyType.Vector4)
            {
                name = GetVariableNameForNode(),
                vector4Value = propColor
            });
`